creator-node/src/snapbackSM/snapbackSM.js

```diff
@@ -347,27 +396,37 @@ class SnapbackSM {
         newReplicaSetSPIds.slice(1) // [secondary1, secondary2]
       )
 
+      response.issuedReconfig = true
+
       // Sync data from existing primary to new secondary
       phase = issueUpdateReplicaSetOpPhases.ENQUEUE_SYNCS
+      await this.enqueueSync({
+        ...baseSyncRequestParams,
+        primaryEndpoint: primary,
+        secondaryEndpoint: newSecondary1
+      })
       await this.enqueueSync({
         ...baseSyncRequestParams,
         primaryEndpoint: primary,
         secondaryEndpoint: newSecondary2
       })
     } catch (e) {
-      errorMsg = `[issueUpdateReplicaSetOp] userId=${userId} wallet=${wallet} failed at phase=${phase} reconfiguring to spIds=[${newReplicaSetSPIds}]: ${e.toString()}\n${e.stack}`
+      const errorMsg = `[issueUpdateReplicaSetOp] userId=${userId} wallet=${wallet} failed at phase=${phase} reconfiguring to spIds=[${newReplicaSetSPIds}]: ${e.toString()}\n${e.stack}`
+      response.errorMsg = errorMsg
+      return response
     }
 
-    return errorMsg
+    return response
   }
 
   /**
+   * Logic to determine the new replica set. Used in reconfig op.
```

Comment:
with so much going on in this function, a more detailed comment here would make this much easier to follow

Reply:
updating comments